modules/cereal/1.3.2/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/USCiLab/cereal/archive/refs/tags/v1.3.2.tar.gz",
+    "integrity": "sha256-FqetmzG6WIDaxV1itdbyQ8PryNRqNRQUnla15+qB+F8=",
+    "strip_prefix": "cereal-1.3.2",
+    "overlay": {
```

Comment:
overlay is only supported in Bazel 7, so please remove "6.x" in presubmit.yml.

Reply:
Got it, thanks!